### DISCUSSION TITLE
Installation on Windows 10

### DISCUSSION THREAD

**Discussion by cecereroberto** [juanmanzanero/fastest-lap#11] [Q&A] [answered]
Good evening everyone, could someone give me the documentation for installing the software on Windows? A thousand thanks

**Comment by juanmanzanero** [comment, marked as answer]
Hello Roberto! The software is not supported in windows yet. It is quite complicated to migrate the framework I am using that depends on many third party libraries. I will work on it on the future but for now it is not possible. Keep posted to this issue: https://github.com/juanmanzanero/fastest-lap/issues/3

**Reply by juanmanzanero** [reply]
(Forgot to say it works on Linux and Mac)

**Reply by Leersmaekers** [reply]
Do you have an update regarding the installation steps on a windows machine? In the past I've built an own laptime simulation with approximately the same toolchain. I remember it took me several months to figure out which combinations of 3rd party libraries could work together. In the end I used Intel (OneApi) MKL with IpOpt. I didnt go through libraries such as Metis and Mumps.

**Reply by juanmanzanero** [reply]
Hello! Yes, I managed to compile on windows using Visual Studio, and uploaded the precompiled binaries on the Release section (https://github.com/juanmanzanero/fastest-lap/releases)

**Reply by Leersmaekers** [reply]
Do you have some documentation on how to compile it for Windows?

**Reply by juanmanzanero** [reply]
Not yet, it involves a few dirty steps as a result of the third party libs and I want to find a cleaner way. The precompiled binaries do not work for you?

**Reply by Leersmaekers** [reply]
Well, I'm currently more interested in learning more about the code behind. Therefore it would be good to be able going through the application with a debugger. My biggest challenge (own code I wrote) had to do with the gradient and Hessian calculations in combination with a numerical solver such as RK or CrankMicholson.

**Reply by juanmanzanero** [reply]
You can still use WSL and debug using GDB, I personally use this method.

**Reply by Leersmaekers** [reply]
Okay, I will try this method first. Which IDE do you prefer for coding in a Linux environment?
